docs/administration/upgrading/guide/automate-upgrades.md

```diff
@@ -120,147 +77,105 @@ Write-Host "Maintenance's response: $maintenanceResponse"
 }
 
-if ($null -eq $versionToDownload -or $versionToDownload -ne $apiInformation.Version)
-{
-    exit 0
-}
-
-Write-Host "Downloading $versionToDownload"
-$filePath = "C:\Temp\Octopus.$versionToDownload-x64.msi"
-$url = "https://download.octopusdeploy.com/octopus/Octopus.$versionToDownload-x64.msi"
-Invoke-RestMethod -Method GET -Uri $url -OutFile $filePath
-
-$OctopusAPIKey = "YOUR API KEY"
-$OctopusURL = "https://yoururl.something.com"
-$IsMaintenanceMode = $true
-
-$header = New-Object "System.Collections.Generic.Dictionary[[String],[String]]"
-$header.Add("X-Octopus-ApiKey", $OctopusAPIKey)
-$header.Add("X-HTTP-Method-Override", "PUT")
-
-$rawRequest = @{
-  Id = "maintenance";
-  IsInMaintenanceMode = $IsMaintenanceMode;  ## Change this to false when you want to take out of maintenance mode
-  Links = @{
-    Self = "/api/maintenanceconfiguration";
-  }
-}
-$jsonRequest = $rawRequest | ConvertTo-Json
+# Finish any remaining tasks and stop the service
+& $installPath\Octopus.Server.exe node --instance="OctopusServer" --drain=true --wait=0
+& $installPath\Octopus.Server.exe service --instance="OctopusServer" --stop
 
-Write-Host "Sending in the request $jsonRequest"
+# Backup database
+$fileName = 'OctopusDeploy_' + (Get-Date -Format FileDateTime) + '.bak'
+sqlcmd -S 'sqlserver.octopus.app' -Q "BACKUP DATABASE [OctopusDeploy] TO DISK = .\$fileName WITH FORMAT;"
 
-$maintenanceUrl = "$OctopusUrl/api/maintenanceconfiguration"
-Write-Host "Setting maintenance mode $maintenanceUrl"
-$maintenanceResponse = Invoke-RestMethod $maintenanceUrl -Headers $header -Method POST -Body $jsonRequest
+# Running the installer
+msiexec /i "${env:TEMP}\$msiFilename" /quiet | Out-Null
 
-Write-Host "Maintenance's response: $maintenanceResponse"
+# Upgrade database and restart service
+& $installPath\Octopus.Server.exe database --instance="OctopusServer" --upgrade
+& $installPath\Octopus.Server.exe service --instance="OctopusServer" --start
+& $installPath\Octopus.Server.exe node --instance="OctopusServer" --drain=false
 
-Set-Location "${env:ProgramFiles}\Octopus Deploy\Octopus" 
-
-& .\octopus.server.exe node --instance="OctopusServer" --drain=true --wait=0
-& .\octopus.server.exe service --instance="OctopusServer" --stop
-
-Write-Output "Installing MSI $filePath" 
-$msiExitCode = (Start-Process -FilePath "msiexec.exe" -ArgumentList "/i $filePath /quiet" -Wait -Passthru).ExitCode 
-Write-Output "Server MSI installer returned exit code $msiExitCode" 
-
-& .\octopus.server.exe database --instance="OctopusServer" --upgrade
-& .\octopus.server.exe service --instance="OctopusServer" --start
-& .\octopus.server.exe node --instance="OctopusServer" --drain=false 
+Remove-Item "${env:TEMP}\$msiFilename"
 ```
 
+Depending on the upgrade, your Octopus Server may still be starting up when the script completes. It will still be in maintenance mode, giving you a chance to login to the UI and verify things are working as expected.
```

Comment:
```suggestion
Depending on the duration of the upgrade, your Octopus Server may still be starting up when the script completes. It will still be in maintenance mode, giving you a chance to log in to the Octopus web portal and verify things are working as expected.
```